apps/meteor/client/views/room/composer/messageBox/MessageBox.tsx

```diff
@@ -45,6 +45,11 @@ import MessageBoxFormattingToolbar from './MessageBoxFormattingToolbar';
 import MessageBoxReplies from './MessageBoxReplies';
 import { useMessageBoxAutoFocus } from './hooks/useMessageBoxAutoFocus';
 import { useMessageBoxPlaceholder } from './hooks/useMessageBoxPlaceholder';
+import fileSize from 'filesize';
+import { e2e } from '../../../../../app/e2e/client';
+import { getFileExtension } from '../../../../../lib/utils/getFileExtension';
+import { Box } from '@rocket.chat/fuselage';
+import FilePreview from '../../modals/FileUploadModal/FilePreview';
```

Comment:
The preview is not something from the modal anymore. I think it's not even necessart to have modal accepting multiple files anymore, so you should move the component to here, close to the place it's used

Reply:
Is it good to move filePreview inside `apps/meteor/client/views/room/composer/messageBox/filePreview/`  folder?

Reply:
I think so

Reply:
done sir